nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py

```diff
@@ -0,0 +1,267 @@
+"""
+Python DAQmx API program:
+	ContGen_IntClk
+
+Author: Andres Quesada
```

Comment:
Everything in this repo should inherit the [LICENSE](https://github.com/ni/nidaqmx-python/blob/master/LICENSE). Let's remove these author/copyright blocks.